usecases/traverser/explorer.go

```diff
@@ -406,6 +407,14 @@ func (e *Explorer) searchResultsToGetResponse(ctx context.Context,
 			additionalProperties["lastUpdateTimeUnix"] = res.Updated
 		}
 
+		replEnabled, err := e.replicationEnabled(params)
```

Comment:
It makes more sense to me to move `replEnabled, err := e.replicationEnabled(params)`outside the loop so that it is called just once.

Reply:
Agreed, thanks